scarf/utils.py

```diff
@@ -249,3 +250,31 @@ def rolling_window(a, w):
         for j in range(m):
             b[i, j] = a[x:y, j].mean()
     return b
+
+
+def permute_in_chunks(size: int, chunk_size: int, seed: int = 42) -> List[np.ndarray]:
```

Comment:
Some suggestions for name
`permute_and_chunk`?
`shuffle_and_chunk`?
`permute_into_chunks`?